components/NavigationDrawer/tests/unit/MDCNavigationDrawerScrollViewTests.m

```diff
@@ -768,4 +770,66 @@ - (void)testFullScreenContentLayoutCalculationsComplete {
   // It has no visible side effects, so as long as this test finishes, it passes
 }
 
+- (void)testSettingElevationInDrawerViewControllerPropogatesToPresentationController {
```

Comment:
Can you add tests for the changes to `presentationTransitionWillBegin`? Either a unit test or snapshot test would be fine. If you remove the `nil` checks there, I don't think they would be necessary.

Reply:
Removed nil checks in code